tests/unit_tests/test_lobbyconnection.py

```diff
@@ -369,6 +370,40 @@ async def test_command_game_join_uid_as_str(
         "mod": "faf",
         "uid": 42,
         "name": "Test Game Name",
+        "game_type": "custom",
+        "init_mode": InitMode.NORMAL_LOBBY.value,
+        "rating_type": "global",
+    }
+    lobbyconnection.send.assert_called_with(expected_reply)
+
+async def test_should_reply_with_game_type_field_on_game_lauch(database,
```

Comment:
Can we format like this please:
```python3
def long_function_name(
    arg1,
    arg2,
    ...
):
    pass
```

Reply:
Isn't this testing exactly the same thing as `test_command_game_join_calls_join_game`? I think it would be more useful to add some tests for the other game types: coop and matchmaker. Maybe it would also be good to add a test of the "integration test" variety in `tests/integration_tests/test_game.py`. I don't think we have one yet that actually checks the contents of the game_launch message. I'd also modify `tests/integration_tests/test_matchmaker.py::test_game_launch_message` to check the game type.